src/App.jsx

```diff
@@ -14,57 +15,59 @@
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [isSelectedGood, setIsSelectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  const handleClear = () => setIsSelectedGood('');
+  const handleSelect = good => {
+    setIsSelectedGood(prev => (prev === good ? '' : good));
+  };
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button data-cy="AddButton" type="button" className="button">
-              +
-            </button>
-          </td>
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {isSelectedGood ? `${isSelectedGood} is selected` : 'No goods selected'}
```

Comment:
The conditional rendering here is correct, but consider using a more descriptive message for when no goods are selected, such as 'Please select a good.'